.eslintrc.js

```diff
@@ -1,34 +1,34 @@
 module.exports = {
-    "env": {
-        "browser": true,
-        "es6": true,
-        "node": true
-    },
-    "extends": "eslint:recommended",
-    "globals": {
-        "Atomics": "readonly",
-        "SharedArrayBuffer": "readonly"
-    },
-    "parserOptions": {
-        "ecmaVersion": 11,
-        "sourceType": "module"
-    },
-    "rules": {
-        "indent": [
-            "error",
-            2
```

Comment:
4

Reply:
We can stay with 2 if you want, I don't care anyway but it's better to use 4 spaces to be along with the standards

Reply:
as u wish